src/Ombi/ClientApp/src/app/components/detailed-card/detailed-card.component.html

```
@@ -1,52 +1,48 @@
-<div id="detailed-{{request.mediaId}}" class="detailed-container" [style.background-image]="background">
-    <div class="row">
-        <div class="col-xl-5 col-lg-5 col-md-5 col-sm-12 posterColumn">
-            <ombi-image (click)="click()" [src]="request.posterPath" [type]="request.type" class="poster" alt="{{request.title}}">
-            </ombi-image>
+<div id="detailed-{{request.mediaId}}" class="detailed-card" (click)="click()">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)